freeipa/src/main/java/com/sequenceiq/freeipa/service/stack/instance/DefaultInstanceGroupProvider.java

```diff
@@ -55,6 +56,11 @@ public Template createDefaultTemplate(CloudPlatform cloudPlatform, String accoun
                     entry(AwsInstanceTemplate.EBS_ENCRYPTION_ENABLED, Boolean.TRUE),
                     entry(InstanceTemplate.VOLUME_ENCRYPTION_KEY_TYPE, EncryptionType.DEFAULT.name()))));
         }
```

Comment:
General note for the class: After fixing the logic below, please be sure to update the UT for `createDefaultTemplate()`.